.gitignore

```diff
@@ -66,3 +66,6 @@ test/types/index.js
 
 # compiled app
 dist
+
```

Comment:
It doesn't like you built your branch based on a checkout of the earlier PR, otherwise the prior edits should have disappeared now that the earlier PR has been merged. Rebuild your branch off of main please? Thx!

Reply:
Hii leona, so this was how i made the changes, 1. pulled from the main branch 2. checkout the feature/01-openapi-spec-inject 3. Created a new branch 4. made changes 5. pushed new branch 6. made a PR to the main branch. 

Do I need to create a new branch off my main branch by checking out main (git checkout main) then creating a new branch off it (git checkout -b feature/03-imporove-sample-json) ? Apologies if I am messing up this PR. i haven't used git and github collaboratively with other people before. 






[12:14](https://co-devs-co.slack.com/archives/D06L81X44CX/p1712920441236509)
From the document in the wayfinder channel, I see that I should have pulled the feature/01-opanapi-spec-injest branch

Reply:
Hi Arinze,

You are suffering (we are all suffering!) from a well-known GitHub issue.

My suggestion is that you do `git pull origin main` in your local branch, make the changes suggested, commit and push back up.

Reply:
Hii Leona, 

I have done that. 

Let me know if it is done right.